src/web/layouts/CommentLayout.tsx

```diff
@@ -231,8 +219,11 @@ export const CommentLayout = ({ CAPI, NAV }: Props) => {
                             sectionUrl={CAPI.sectionUrl}
                             guardianBaseURL={CAPI.guardianBaseURL}
                             pillar={CAPI.pillar}
-                            badge={GE2019Badge}
+                            badge={CAPI.badge}
                             inLeftCol={true}
+                            frontendAssetsFullURL={
```

Comment:
How easy would it be to prefix the badge imageUrl with `frontendAssetsFullURL` higher up, so we don't need to pass both down?

Maybe here, in the layout, but better might be in frontend?

Reply:
Good shout, did that! https://github.com/guardian/frontend/pull/22263/files#diff-490210a7e352de3ddfc2d98c1db606dfR585